subprojects/core/src/test/groovy/org/gradle/internal/scan/config/BuildScanConfigManagerTest.groovy

```diff
@@ -57,43 +59,92 @@ class BuildScanConfigManagerTest extends Specification {
         with(config()) {
             !enabled
             disabled
+            attributes.is this.attributes
         }
     }
 
     def "throws if plugin version is too old"() {
         when:
-        pluginVersionNumber = "0.9"
-        config()
+        config("1.7")
 
         then:
         thrown UnsupportedBuildScanPluginVersionException
 
         when:
-        pluginVersionNumber = "1.1"
-        config()
+        config("1.9")
 
         then:
         notThrown UnsupportedBuildScanPluginVersionException
 
         when:
-        pluginVersionNumber = "1.0-TIMESTAMP"
-        config()
+        config("1.8-TIMESTAMP")
 
         then:
         notThrown UnsupportedBuildScanPluginVersionException
     }
 
+    def "throws if has vcs mappings and plugin version is too old"() {
+        given:
+        attributes.isRootProjectHasVcsMappings() >> true
+
+        when:
+        config("1.9")
+
+        then:
+        thrown UnsupportedBuildScanPluginVersionException
+    }
+
+    def "does not throw if has vcs mappings and plugin version 1.10 or newer"() {
```

Comment:
this test doesn't test version 1.10 as described in the title

Reply:
Done.